Makefile

```diff
@@ -0,0 +1,91 @@
+include .env
+BINARY := k8s-spot-rescheduler
+VERSION := $(shell git describe --always --dirty --tags 2>/dev/null || echo "undefined")
+
+RED := \033[31m
+GREEN := \033[32m
+NC := \033[0m
+
+IMG ?= quay.io/pusher/k8s-spot-rescheduler
+
+.NOTPARALLEL:
+
+.PHONY: all
+all: test build
```

Comment:
I'd suggest adding `distclean` to the dependencies to ensure a naive `make all` always works.